modules/querying/pages/func/query-user-defined-functions.adoc

```diff
@@ -176,3 +176,37 @@ GSQL > RUN QUERY rngExample()
   "results": [{"rng()": 0.51352}]
 }
 ----
+=== Using GitHub to Store UDFs
+
+An alternative method is to connect GSQL with a GitHub repository for access to ExprFunctions and ExprUtil. This enables a simpler editing workflow rather than repeatedly using `PUT` to upload the file for every change.
+
+In fact, GSQL will retrieve user source code files from GitHub before files uploaded via `PUT`, so long as the files exist.
+
+The gadmin configuration parameters are as follows:
+
+[source]
+----
+GSQL.GithubUserAcessToken # the credential, or "anonymous", empty means not using github
+GSQL.GithubRepository     # e.g. tigergraph/ecosys
+GSQL.GithubBranch         # optional, o/w use "master" branch, e.g. demo_github
+GSQL.GithubPath           # path to the directory in the github that has TokenBank.cpp, ExprFunctions.hpp, ExprUtil.hpp, e.g. sample_code/src
+GSQL.GithubUrl            # optional, used for github enterprise, e.g. https://api.github.com
+----
+
+Run the following `gadmin config set` commands to give GSQL access to GitHub.
```

Comment:
The commands that follow are only an example configuration. It can be a little misleading to tell users to just run these commands as they might be interpreting it as running the commands as they are, especially if you haven't thoroughly explained what each parameter does. 

A better phrasing would be:

Use the `gadmin config set` (link to gadmin config here) command to configure the value of the aforementioned parameters to connect GSQL to your GitHub repository hosting your UDF file. 
Below is an example configuration.